forwards_paper.tex

```diff
@@ -726,38 +699,119 @@ \subsection*{Simuation methods}
 \subsection*{Simplification}
 
 Here we describe the simplification algorithm.
-\plr{I am trying it out to see what it looks like.}
+The flow is close to our implementation,
+but the description below uses set operations,
+while in the implementation,
+all ancestors are maintained as linked lists of segments
+ordered by left endpoint.
```

Comment:
I am trying to follow the taocp style, but could write this more compactly as a sequence of operators on sets of sets.  That would move it further away from the actual implementation, but make it easier to understand.

Reply:
Efficiency of understanding is definitely the most important here. Looking at the algorithm, I wondered if it's not more suited to an enumerate environment (or rather, 3). We can then use some more text and set notation, without getting too worried about the exact details. There's not a lot of flow control in it (mostly going from step-to-step), so it should be OK I think. 

Although, we do lose the nice simple step labels then.